client/db/types.go

```diff
@@ -99,21 +111,25 @@ func DecodeAccountInfo(b []byte) (*AccountInfo, error) {
 }
 
 func decodeAccountInfo_v0(pushes [][]byte) (*AccountInfo, error) {
-	if len(pushes) != 5 {
-		return nil, fmt.Errorf("decodeAccountInfo: expected 5 data pushes, got %d", len(pushes))
+	if len(pushes) != 7 {
+		return nil, fmt.Errorf("decodeAccountInfo: expected 7 data pushes, got %d", len(pushes))
 	}
 	hostB, keyB, dexB := pushes[0], pushes[1], pushes[2]
 	coinB, certB := pushes[3], pushes[4]
 	pk, err := secp256k1.ParsePubKey(dexB)
 	if err != nil {
 		return nil, err
 	}
+	var id [blake256.Size]byte
```

Comment:
```go
var aid account.AccountID
copy(aid[:], pushes[5])
```

The size of the array being the size of a blake256 hash is the business of the `account` package.

Also, it would be good to verify that the length of `pushes[5]` is `len(aid)` before copying.